templates/_base.text.twig

```diff
@@ -4,3 +4,8 @@
     Big Give Team
     BigGive.org
 {% endblock %}
+{# Included by default - override with an empty block in templates where this is not relavent. #}
+{% block cardDetailsHeldByStripe %}
+
+    Your card details are held securely by Stripe. We do not have access to your card details.
+{% endblock %}
```

Comment:
Adding this here - I'm assuming it was an oversight that we had this message in HTML email parts but in plain text email parts.